src/sql/workbench/contrib/notebook/browser/cellToolbarActions.ts

```diff
@@ -151,10 +151,11 @@ export class CellToggleMoreActions {
 
 	public onInit(elementRef: HTMLElement, context: CellContext) {
 		this._moreActionsElement = elementRef;
+		this._moreActionsElement.setAttribute('aria-haspopup', 'menu');
```

Comment:
Shouldn't this still have the role:button attribute?

Reply:
I got this value from accessibility folks in the office hours meeting that I attended last week. 
[8/27 2:11 PM] Sarah Higley:     button aria-haspopup="menu"

Adding that is enough, we don't need to add role:button
![image](https://user-images.githubusercontent.com/12754347/91688013-dcd4de00-eb15-11ea-8ea2-c1fb05a76ee2.png)

Reply:
As per Maddy, we do not need to change this.